apis/user_search.yaml

```
@@ -0,0 +1,852 @@
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?order={order phrase}:
+    get:
+      summary: "Sort Users"
+      description: >
+        User results from this API can be sorted by a number of fields using the `order` param. If not specified, the
+        results are sorted by relevance. If a sort field is specified that the search service does not
+        recognize, the query will fail. For all accepted sort values, either `ASC` or `DESC` can optionally
+        be appended with a leading space. Note that spaces must be URL-escaped with `+` or `%20`.
+
+
+        It is possible for search results to have missing values for some of these sort fields (`last_authenticated_at`
+        for example). Any assets missing a value altogether for the field being sorted on will show up at the end of the
+        results list.
+
+
+        The accepted values for the `order` parameter are:
+          - `relevance` (default)
+          - `email`
+          - `screen_name`
+          - `role_name` (also requires the `domain` param)
+          - `last_authenticated_at` (also requires the `domain` param)
+
+        If using either of the last 2 values, please read the "Domain specific params" on the
+        [overview page](#?route=overview) to avoid common errors.
+
+        #### Examples
+          - `?order=relevance%20DESC`
+          - `?order=email+ASC`
+          - `?order=screen_name`
+          - `?order=role_name%20ASC&domain=data.ny.gov`
+          - `?order=last_authenticated_at&domain=data.ny.gov`
+
+      parameters:
+        - $ref: '#/components/parameters/order'
+        - $ref: '#/components/parameters/domain'
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SearchResponse'
+        "400":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequestError'
+        "401":
+          description: ""
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+  /catalog/v1/users?offset={number}&limit={number}:
+    get:
+      summary: "Page through Users"
+      description: >
+        This API allows pagination of results via 2 parameters:
+         1. `offset` - the number of results already paged through. Results numbered lower than this given number are
+                       skipped and not returned.
+         2. `limit` - the number of results to include.
+
+
+        For example `?offset=0&limit=10` followed by `?offset=10&limit=10` would find the first 10 users, then the next
+        10 users. By default, limit=100 and offset=0. If the sum of the `offset` and `limit` parameters is more than
```

```suggestion
        10 users. By default, `limit=100` and `offset=0`. If the sum of the `offset` and `limit` parameters is more than
```
maybe worth highlighting the defaults with backticks